src/components/EmojiPicker.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from "prop-types";
+import React from "react";
+import ReactDOM from "react-dom";
+import "emoji-mart/css/emoji-mart.css";
+import { Picker } from "emoji-mart";
+import IconButton from "material-ui/IconButton";
+import EditorInsertEmoticon from "material-ui/svg-icons/editor/insert-emoticon";
+
+class PickerWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      x: 0,
+      y: 0
+    };
+
+    this.handleClickOutside = this.handleClickOutside.bind(this);
+  }
+
+  componentDidMount() {
+    const bounds = ReactDOM.findDOMNode(this.picker).getBoundingClientRect();
```

Comment:
rather than go so low-level like this could we use:
`import Popover from "material-ui/Popover";`
See src/components/AssignmentTexter/Controls.jsx for an example.